src/ui/Card/Card.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import { Meta, StoryObj } from '@storybook/react'
+
+import { Card } from './Card'
+
+const meta: Meta<typeof Card> = {
+  title: 'Components/Card',
+  component: Card,
+} as Meta
+export default meta
+
+type Story = StoryObj<typeof Card>
+
+export const Default: Story = {
+  render: () => (
+    <Card>
+      <Card.Content>
+        Here is the Card component, you are free to render anything you would
+        like here.
+      </Card.Content>
+    </Card>
+  ),
+}
+
+export const CardWithHeader: Story = {
+  render: () => (
+    <Card>
+      <Card.Header>
+        <Card.Title>A header can have a title.</Card.Title>
+        <Card.Description>And it can have a description.</Card.Description>
+      </Card.Header>
+      <Card.Content>
+        The header will place a border between it and the main Card content.
+      </Card.Content>
+    </Card>
+  ),
+}
+
+export const CardWithHeaderVariant: Story = {
+  render: () => (
+    <Card>
+      <Card.Header>
+        <Card.Title>A Title</Card.Title>
+        <Card.Title size="base">A Smaller Title</Card.Title>
+      </Card.Header>
+      <Card.Content>
+        A smaller title can be used by setting the{' '}
+        <code className="bg-ds-gray-secondary">size</code> variant of{' '}
+        <code className="bg-ds-gray-secondary">Card.Title</code>.
+      </Card.Content>
+    </Card>
+  ),
+}
```

Comment:
I would recommend tweaking this story a bit, and allow users to play around with the different variants on their own so they don't have to jump between stories, you can see an example here: https://github.com/codecov/gazebo/blob/main/src/ui/TopBanner/TopBanner.stories.tsx#L31-L44